packages/fx-core/src/core/middleware/utils/appYmlGenerator.ts

```diff
@@ -50,6 +52,8 @@ export class AppYmlGenerator extends BaseAppYmlGenerator {
       isWebAppBot: false,
       isTypescript: false,
       defaultFunctionName: undefined,
+      defaultEnvironmentFolder: undefined,
```

Comment:
In generator, the key name environmentFolder is better than defaultEnvironmentFolder.
Actually, we will refine it to use another folder when there is existing user's env folder in project

Reply:
changed